server.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const app = express();
+const db = require("./database/db.js");
+
+app.use("/", express.static(__dirname + "/client"));
+
+app.get("/get", (req, res) => {
+  if (req.query.id === undefined) {
+    db.getAllProducts(data => {
+      res.send(data);
+    });
+  } else {
+    db.getProduct(req.query.id, data => {
+      res.send(data);
+    });
+  }
+});
+
+app.listen("9000", () => {
```

Comment:
Use process.env.PORT environmental variable for port for easier deployment and integration, lookup dotenv npm package